src/main/protobuf/com/rawlabs/protocol/das/v1/services/tables_service.proto

```diff
@@ -110,6 +110,10 @@ message ExecuteTableRequest {
     com.rawlabs.protocol.das.v1.tables.TableId table_id = 2;
     com.rawlabs.protocol.das.v1.query.Query query = 3;
     optional string plan_id = 4;
+    // Maximum size in bytes for each batch of query results.
+    uint32 max_batch_size_bytes = 5;
+    // Maximum allowable age of an existing cache to consider for reuse.
+    uint32 max_cache_age_seconds = 6;
```

Comment:
This one should be optional, so we don't reuse? Also, isn't there a better type - one of Google proto types perhaps - to represent durations? I wonder if seconds is good enough resolution; I could see it being lower than a second.